src/parsers/poscar.js

```diff
@@ -63,8 +64,22 @@ export function atomsCount(poscarFileContent) {
     return atomsLine.map((x) => parseInt(x, 10)).reduce((a, b) => a + b);
 }
 
+/**
+ * Function returns the string on the first line of POSCAR file content. Generally the first line contains text
+ * containing the name of the structure.
+ * @param {String} fileContent
+ * @param {String} failoverName
+ * @returns {String}
+ */
+export function getNameFromContents(fileContent, failoverName = "material") {
+    const nameFromContent = getLineFromContent(fileContent, 0);
+    if (nameFromContent) return nameFromContent;
```

Comment:
I think this `if` condition will always be true for files with empty lines too - when the line contains only spaces, for example. We need to be careful with the REGEX in `/\r?\n/` and or trim all trailing and beginning whitespace from the line.